src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number,
+  userId: number,
+  title: string,
+  completed: boolean,
+  user: User | null,
```

Comment:
It's better to use [an optional property](https://www.typescriptlang.org/docs/handbook/2/objects.html#optional-properties) instead of using User | null.